Mesh_3/include/CGAL/IO/File_medit.h

```diff
@@ -815,27 +815,28 @@ output_to_medit(std::ostream& os,
        fit != c3t3.facets_in_complex_end();
        ++fit)
   {
-    for (int i=0; i<4; i++)
-    {
-      if (i != fit->second)
-      {
-        const Vertex_handle& vh = (*fit).first->vertex(i);
-        os << V[vh] << ' ';
-      }
-    }
+    typename C3T3::Facet f = (*fit);
+    
+    // Apply priority among subdomains, to get consistent facet orientation per subdomain-pair interface.
+    if (f.first->subdomain_index() < f.first->neighbor(f.second)->subdomain_index())
+      f = tr.mirror_facet(f);
```

Comment:
Hi @goran-w,

Currently the requirements on [`MeshDomain_3::Subdomain_index`] do not include [LessThanComparable] but only [EqualityComparable]. You patch adds the requirements on [LessThanComparable] (the `<` operator).

[`MeshDomain_3::Subdomain_index`]: https://doc.cgal.org/4.13/Mesh_3/classMeshDomain__3.html#a30d50e8d38bee7db92b4ec0f813e8e53

[LessThanComparable]: https://en.cppreference.com/w/cpp/named_req/LessThanComparable
[EqualityComparable]: https://en.cppreference.com/w/cpp/named_req/EqualityComparable

Reply:
Well, on the other hand we currently assume, in the code of `File_medit.h`, that the type is convertible to `int`, so in practice only integral types are accepted.